main.js

```diff
@@ -18,21 +17,24 @@ function newElement() {
     }
     document.getElementById("txt").value = ""; 
 
-  //--tạo ra icon thùng rác--
+    //--tạo ra icon thùng rác--
     let span = document.createElement("span"); //tạo ra 1 span
     let i = document.createElement("i"); //tạo ra 1 thher i
     i.className = ('far fa-trash-alt'); // thêm 1 className cho i
     span.className = "close"; // thêm class cho span
     span.appendChild(i); // nối i vào span
     li.appendChild(span); //nối span vào li
-  
-    //dele
+    span.addEventListener('click', deleElement); // thêm sk vào nút xóa
+    
+  }
+
+  function deleElement(){
```

Comment:
Change the name of the function, and make it more clear. For example: `deleteTask`

Reply:
Done